src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def user_filesystem(tmp_path):
+    directory = Path(tmp_path)
+    os.chdir(directory)
+
+    existing_dir = Path(tmp_path).resolve() / "existing_dir"
+    existing_dir.mkdir(parents=True, exist_ok=True)
+
+    existing_file = Path(tmp_path).resolve() / "existing_file.py"
+    existing_file.touch()
+
+    input_directory = Path(tmp_path).resolve() / "input_directory"
+    input_directory.mkdir(parents=True, exist_ok=True)
+
+    with open(os.path.join(input_directory, "good_data.chi"), "w") as f:
+        f.write("1 2 \n 3 4 \n 5 6 \n 7 8")
+    with open(os.path.join(input_directory, "good_data.xy"), "w") as f:
```

Comment:
this seems to be the same file contents as `good_data.chi`.  Actually we probably don't need this for testing, but it may be helpful for documentation.   However, the distinction in general is that chi files can have a header, then a data-block, so to make it clearer to people the intention of a test, leave this xy file as is but make the chi file look more like a real chi file?